cpp/turbodbc_arrow/Library/src/arrow_result_set.cpp

```diff
@@ -242,10 +247,23 @@ Status arrow_result_set::fetch_all_native(std::shared_ptr<arrow::Table>* out)
     return Status::OK();
 }
 
+
+pybind11::object arrow_result_set::fetch_next_batch()
+{
+    std::shared_ptr<arrow::Table> table;
+    if (not fetch_all_native(&table, true).ok()) {
```

Comment:
Constants `fetch_single_batch = true` and `fetch_all_batches = false` could be used here and below to clear up the meaning of `true` and `false`. If you find a way to avoid the switch in the call (e.g. by composing fetch all in terms of fetch single batch), that would be even better.